tests/transforms/test_convert_rois.py

```diff
@@ -38,7 +38,7 @@ def test_output_schema_element():
 
 @pytest.mark.parametrize("s2p_stat_fixture, ophys_movie_fixture, "
                          "motion_correction_fixture, "
-                         "longest_edge_threshold, expected_rois",
+                         "aspect_threshold, expected_rois",
```

Comment:
Since there was a bug related to the edge calculation in the original code which some new tests caught, could you also include a mask that just has one pixel?

Reply:
Will add a single-pixel test

Reply:
added singleton and empty test for the updated function.